apps/condo/lang/en/en.json

```diff
@@ -642,6 +642,7 @@
   "pages.condo.meter.index.PageTitle": "Meters log",
   "pages.condo.meter.index.EmptyList.header": "The list is empty",
   "pages.condo.meter.index.CreateMeterButtonLabel": "Add new meter",
+  "pages.condo.meter.index.CreateMeterReadingsButtonLabel": "Take meter readings",
```

Comment:
Submit?

Reply:
Translation is same as for button in side menu